shared/menubar/index.desktop.js

```diff
@@ -277,133 +203,132 @@ class MenubarRender extends Component<Props, State> {
-    }}
-  />
-)
-
 const BadgeIcon = ({
   tab,
   countMap,
   openApp,
 }: {
-  tab: Tab,
+  tab: Tabs.Tab,
   countMap: Object,
   openApp: (tab: ?string) => void,
 }) => {
   const count = countMap[tab]
 
-  if (tab === devicesTab && !count) {
+  if (tab === Tabs.devicesTab && !count) {
     return null
   }
 
-  const iconMap: {[key: Tab]: IconType} = {
-    [peopleTab]: 'iconfont-nav-people',
-    [chatTab]: 'iconfont-nav-chat',
-    [devicesTab]: 'iconfont-nav-devices',
-    [fsTab]: 'iconfont-nav-files',
-    [teamsTab]: 'iconfont-nav-teams',
+  const iconMap: {[key: Tabs.Tab]: Kb.IconType} = {
+    [Tabs.peopleTab]: 'iconfont-nav-people',
+    [Tabs.chatTab]: 'iconfont-nav-chat',
+    [Tabs.devicesTab]: 'iconfont-nav-devices',
+    [Tabs.fsTab]: 'iconfont-nav-files',
+    [Tabs.teamsTab]: 'iconfont-nav-teams',
   }
-  const iconType: ?IconType = iconMap[tab]
+  const iconType: ?Kb.IconType = iconMap[tab]
 
   if (!iconType) {
     return null
   }
 
   return (
-    <Box
-      style={{...desktopStyles.clickable, marginLeft: 7, marginRight: 7, position: 'relative'}}
+    <Kb.Box
+      style={{...Styles.desktopStyles.clickable, marginLeft: 7, marginRight: 7, position: 'relative'}}
       onClick={() => openApp(tab)}
     >
-      <Icon color={globalColors.darkBlue4} hoverColor={globalColors.black_75} fontSize={22} type={iconType} />
-      {!!count && <Badge badgeNumber={count} badgeStyle={{position: 'absolute', top: -6, right: -8}} />}
-    </Box>
+      <Kb.Icon color={Styles.globalColors.darkBlue4} hoverColor={Styles.globalColors.black_75} fontSize={22} type={iconType} />
+      {!!count && <Kb.Badge badgeNumber={count} badgeStyle={{position: 'absolute', top: -6, right: -8}} />}
+    </Kb.Box>
   )
 }
 
-const borderRadius = 4
-
-const stylesContainer = {
-  ...globalStyles.flexBoxColumn,
-  flex: 1,
-  position: 'relative',
-  marginTop: 13,
-  borderTopLeftRadius: borderRadius,
-  borderTopRightRadius: borderRadius,
-}
-
-const stylesTopRow = {
-  ...globalStyles.flexBoxRow,
-  alignItems: 'center',
-  backgroundColor: globalColors.darkBlue2,
-  flex: 1,
-  minHeight: 40,
-  maxHeight: 40,
-  paddingLeft: 8,
-  paddingRight: 8,
-  borderTopLeftRadius: borderRadius,
-  borderTopRightRadius: borderRadius,
-}
-
-const stylesPrivate = {
-  container: {
-    ...stylesContainer,
-    backgroundColor: globalColors.darkBlue,
-  },
-}
-
-const stylesPublic = {
-  container: {
-    ...stylesContainer,
-    backgroundColor: globalColors.white,
-  },
-}
-
-const stylesLogo = {
-  alignSelf: 'center',
-  marginBottom: 12,
-}
+const styles = Styles.styleSheetCreate({
+  widgetContainer: Styles.platformStyles({
+    common: {
+      ...Styles.globalStyles.flexBoxColumn,
+      flex: 1,
+      position: 'relative',
+      marginTop: isDarwin ? 13 : 0,
+      borderTopLeftRadius: Styles.globalMargins.xtiny,
+      borderTopRightRadius: Styles.globalMargins.xtiny,
+      backgroundColor: Styles.globalColors.darkBlue,
+    },
+  }),
```

Comment:
These and some below only have `common`, so why not just do this instead?

```
widgetContainer: {
    ...Styles.globalStyles.flexBoxColumn,
    ... ...
},
```

Reply:
Mainly because of Flow.